### ISSUE TITLE
Rustdoc should note default-feature in item descriptions

### ISSUE THREAD

**Issue by sinistersnare** [rust-lang/rust#105669]
Currently, if an item is feature-gated (Such as Axum's [Query struct](https://docs.rs/axum/latest/axum/extract/struct.Query.html)), it is noted at the top. That is awesome!

I think that if that feature is marked as a default feature, that should be noted too. As is the wording: "Available on crate feature FEATURE only." implies it wont work by default.

**Comment by RodBurman** [comment]
Agreed it would be clearer in these cases for the documentation to say something like: "Available on crate feature FEATURE only. Feature FEATURE is enabled by default."; thus highlighting that configuration changes are only required if the user wants to disable the feature for some reason.
